contracts/src/v0.1/L1EndpointRequestResponse.sol

```diff
@@ -0,0 +1,301 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+import "./RequestResponseConsumerBase.sol";
+import "./L1EndpointBase.sol";
+import "./RequestResponseConsumerFulfill.sol";
+
+abstract contract L1EndpointRequestResponse is
+    RequestResponseConsumerBase,
```

Comment:
Can you be more specific on this please, I'm not sure I understand what you mean

Reply:
This is just visual suggestion. Currently, the order of inherited parent classes is as 

```
RequestResponseConsumerBase,
L1EndpointBase,
RequestResponseConsumerFulfillUint128,
RequestResponseConsumerFulfillInt256,
RequestResponseConsumerFulfillBool,
RequestResponseConsumerFulfillString,
RequestResponseConsumerFulfillBytes32,
RequestResponseConsumerFulfillBytes
```

My proposal is 

```
L1EndpointBase,
RequestResponseConsumerBase,
RequestResponseConsumerFulfillUint128,
RequestResponseConsumerFulfillInt256,
RequestResponseConsumerFulfillBool,
RequestResponseConsumerFulfillString,
RequestResponseConsumerFulfillBytes32,
RequestResponseConsumerFulfillBytes
```

because it just looks better, not too important.